app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.post = @post
+    @comment.author = current_user
+
+    if @comment.save
+      redirect_to "/users/#{current_user.id}/comments", notice: 'Comment was successfully created.'
+    else
+      render 'posts/show'
+    end
+  end
```

Comment:
- [ ] Please create a `new` method also as you created `create` method and add a route too.

Reply:
This is the method I used in creating the comments, and I don't think it's necessary to add a new method and route as long as the app is working as required, Plus this is not part of the requirements given. Thank you.